docs/content/modeling/testing-models.mdx

```diff
@@ -144,6 +145,68 @@ The following verifies the expected results using the `list_objects` option in <
 ```
 The example above checks that `user:anne` has access to the `organization:acme` as a member and is not an admin of any organization. It also checks that `user:peter`, given the current time is February 1st 2024, 0:10 AM, is not related to any organization as a member, but is related to `organization:acme` as an admin.
 
+## Write List Users tests
+
+:::caution Warning
+ListUsers is currently in an experimental release. Read [the announcement](https://openfga.dev/blog/list-users-announcement) for more information.
+:::
+
+List users tests verify the results of the [list-users API](../getting-started/perform-list-users.mdx) to validate the users who or do not have access to an object
+
+Each list users verification has the following structure:
+
+| Object  | Description |
+| -------- | -------- | 
+|`object` | The object to list users for |
+|`user_filter` | Specifies the type or userset to filter with |
+|`user_filter.type` | The specific type to filter with |
+|`user_filter.relation` | The specific relation to filter with (optional) |
+|`context` | A set of tests for contextual parameters used to evaluate [conditions](./conditions.mdx)|
+|`assertions` | A list of assertions to make |
+|`<relation>` | The name of the relation you want to verify |
+|`<relation>.users` | The users who should have the stated relation to the object |
+|`<relation>.excluded_users` | The users who should have explicitly not have access to the object due to negation in the model |
```

Comment:
I know we use a different term than "public typed wildcard" but think its worth mentioning here.

```suggestion
|`<relation>.excluded_users` | The users who should have explicitly not have access to the object due to evaluations of typed wildcards and negation |
```